.github/workflows/e2e.yml

```diff
@@ -240,15 +244,8 @@ jobs:
           node-version: 18
 
       - uses: actions/setup-java@v4
-        if: ${{ matrix.rn-version == '0.65.3' }}
-        with:
-          java-version: '11'
-          distribution: 'adopt'
-
-      - uses: actions/setup-java@v4
-        if: ${{ matrix.rn-version != '0.65.3' }}
         with:
-          java-version: '17'
+          java-version: ${{ matrix.rn-version == '0.65.3' && '11' || '17' }}
```

Comment:
Thanks, this is super neat, I didn't know the conditions can be used like this in the action's yml.